client/src/components/MapSection.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import "./MapSection.css";
+import dotMapImage from "../images/dot-map.png";
+import ExpandIcon from "../images/ExpandIcon.svg";
+
+function MapSection() {
+	const [isFullScreen, setFullScreen] = useState(false);
+
+	const toggleFullScreen = () => {
+		setFullScreen(!isFullScreen);
+	};
+
+	const handleKeyDown = (e) => {
+		if (e.key === "Enter") {
+			toggleFullScreen();
+		}
+	};
+
+return (
+	<div className={`map-section ${isFullScreen ? "fullscreen" : ""}`}>
+		<div className="help-text">
+			Click on locations to learn more about local rhythms
+		</div>
+		{isFullScreen ? (
+			<div>
+				<div
+					className="exit-fullscreen-button"
+					onClick={toggleFullScreen}
+					//added this part to prevent Eslint rule error and also for accessability
+					onKeyDown={handleKeyDown}
+					tabIndex={0}
+					role="button"
+					aria-pressed={isFullScreen}
+				>
+					Click to exit full screen mode
+				</div>
+				<img src={dotMapImage} alt="Dot Map" />
```

Comment:
It doesn't look like the size of the map changes at all, but otherwise the functionality looks good. We will need to use an SVG for the full version, but this works

Reply:
Scratch that - we need the image to pop up in a modal that overlays the screen. Right now the image stays behind whatever CSS pops up, the image isn't re-rendered